app/main.py

```diff
@@ -1 +1,34 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive = []
```

Comment:
Using a mutable class variable like `alive` can lead to unexpected behavior if instances are modified. Consider using a different design pattern, such as a class method to manage the list of alive animals.